ui/test/services/bom_bar_client_test.dart

```diff
@@ -160,6 +160,34 @@ void main() {
         });
       });
 
+      group('Find obligations for dependencies of a project', () {
+        test('finds obligations per dependency', () async {
+          server.respondJson({
+            'obligation': [
```

Comment:
The value is not actually an identifier-like "obligation", but it is the obligation _description_. (You might want to choose the constant name accordingly.)